### PR TITLE
Change log added

### PR DESCRIPTION
**Pull request by Madhan-SWE**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Added change log for releases.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: CHANGELOG-0.x.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# v0.1.0
+[Documentation](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/blob/v0.1.0/README.md)
+## Downloads for v0.1.0
+
+filename  | sha512 hash
+--------- | ------------
+[v0.1.0.zip](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/archive/refs/tags/v0.1.0.zip) | `ced606ab0248239bfcda9592e58c526d1ce14cf431552b6550ff42c16096a486e19cb0b65f408a0caf8afa78f266f845459e6f530150d8683eda4cf2d29bb5d8`
+[v0.1.0.tar.gz](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/archive/refs/tags/v0.1.0.tar.gz) | `0375378825bff972cb9c50d5d155de1686cd0d5d2ba48464c6447c17433bd28e0bad2eefd391a5e94fdeb33acdd4000b51abf1c261a1d65d0c7dca33fe652906`
+
```
Comment (mkumatag) [inline CHANGELOG-0.x.md:9]: This is not needed as this is available in the release assets section,

File: CHANGELOG-0.x.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* Unit tests added ([#80](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/80), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* Verify volume details added ([#83](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/83), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* Contributing and code of conduct ([#84](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/84), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* Hack utility added ([#86](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/86), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* Node PublishVolume tests added ([#93](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/93), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* Volume lock tests added for controller and node plugins ([#94](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/94), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* PR Template added ([#102](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/102), [@Madhan-SWE](https://github.com/Madhan-SWE))
```
Comment (mkumatag) [inline CHANGELOG-0.x.md:32]: not needed

File: CHANGELOG-0.x.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+[v0.1.0.zip](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/archive/refs/tags/v0.1.0.zip) | `ced606ab0248239bfcda9592e58c526d1ce14cf431552b6550ff42c16096a486e19cb0b65f408a0caf8afa78f266f845459e6f530150d8683eda4cf2d29bb5d8`
+[v0.1.0.tar.gz](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/archive/refs/tags/v0.1.0.tar.gz) | `0375378825bff972cb9c50d5d155de1686cd0d5d2ba48464c6447c17433bd28e0bad2eefd391a5e94fdeb33acdd4000b51abf1c261a1d65d0c7dca33fe652906`
+
+## Changelog since initial commit
+### Notable changes
+* Remove PVM Instance Id from deployment file ([#2](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/2), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* Volume resize functionality added ([#12](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/12), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* Github actions added ([#21](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/21), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* Create OWNERS ([#22](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/22), [@Madhan-SWE](https://github.com/Madhan-SWE))
```
[the remark this answers is not in the document]
Comment (Madhan-SWE) [reply CHANGELOG-0.x.md:15]: can I delete all 4 lines?

File: CHANGELOG-0.x.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+* Verify volume details added ([#83](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/83), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* Contributing and code of conduct ([#84](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/84), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* Hack utility added ([#86](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/86), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* Node PublishVolume tests added ([#93](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/93), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* Volume lock tests added for controller and node plugins ([#94](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/94), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* PR Template added ([#102](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/102), [@Madhan-SWE](https://github.com/Madhan-SWE))
+* Issue template added ([#103](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/pull/103), [@Madhan-SWE](https://github.com/Madhan-SWE))
```
[the remark this answers is not in the document]
Comment (Madhan-SWE) [reply CHANGELOG-0.x.md:33]: can I delete all 4 lines?

File: CHANGELOG-0.x.md (hunk 2 above)
Comment (mkumatag) [reply CHANGELOG-0.x.md:32]: only `PR Template added`

File: CHANGELOG-0.x.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (mkumatag) [reply CHANGELOG-0.x.md:33]: `Issue Template`

File: CHANGELOG-0.x.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (mkumatag) [reply CHANGELOG-0.x.md:15]: `Create OWNERS`

File: CHANGELOG-0.x.md (hunk 1 above)
Comment (Madhan-SWE) [reply CHANGELOG-0.x.md]: Fixed.

File: CHANGELOG-0.x.md (hunk 2 above)
Comment (Madhan-SWE) [reply CHANGELOG-0.x.md]: Fixed

File: CHANGELOG-0.x.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Madhan-SWE) [reply CHANGELOG-0.x.md]: Fixed

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve

**Pull request merged**